.github/workflows/check.yml

```diff
@@ -155,3 +157,32 @@ jobs:
         run: rustup component add rust-src
       - name: Run tests with sanitizer
         run: RUSTFLAGS="-Z sanitizer=${{ matrix.sanitizer }} -Z sanitizer-memory-track-origins" cargo test -Z build-std --target $TARGET --no-default-features --features "cli web-app real-world-infra test-fixture descriptive-gate"
+
+  coverage:
+    name: Measure coverage
+    runs-on: ubuntu-latest
+    env:
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          components: llvm-tools
+
+      - name: Install cargo-llvm-cov
+        run: cargo install cargo-llvm-cov
+
+      - name: Measure coverage
+        run: ./scripts/coverage-ci --lcov --output-path ipa.cov
+
+      - name: Report to codecov.io
+        uses: codecov/codecov-action@v3
+        with:
+          file: ipa.cov
+          fail_ci_if_error: false
```

Comment:
```suggestion
          fail_ci_if_error: false
          token: ${{ secrets.CODECOV_TOKEN }}
```

Reply:
Indeed, it is even marked as [required](https://github.com/codecov/codecov-action?tab=readme-ov-file#arguments)

Reply:
In practice, it's not, but I think that they must fall back to env.